pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -392,6 +397,44 @@ func fixLibvirtCrcNetworkActive() (bool, error) {
 	return true, nil
 }
 
+func checkLibvirtCrcBridgePermissions() (bool, error) {
+	logging.Debug("Checking if 'crc' bridge has appropriate permissions setup")
+	configPath, err := crcos.GetFirstExistentPath(qemuBridgeConfigPaths[:])
+	if err != nil {
+		return false, fmt.Errorf("Failed to find Qemu bridge configuration file: %s", err)
+	}
+
+	config, err := ioutil.ReadFile(filepath.Clean(configPath))
+	if err != nil {
+		return false, fmt.Errorf("Failed to read %s: %v", configPath, err)
+	}
+	regex := regexp.MustCompile(fmt.Sprintf("(\n|^)%s(\n|$)", qemuBridgeConfig))
```

Comment:
Actually it seems the proper syntax for that would be some use of `(?m)` for `^` and `$` to match beginning/end of line in addition to beginning/end of string

> m              multi-line mode: ^ and $ match begin/end line in addition to begin/end text (default false)

Reply:
`go doc regexp/syntax`

Reply:
Thanks. fixed and also added handling of any whitespace.